### PR TITLE
Update shockbs.json, add cname record

### PR DESCRIPTION
**Pull request by shockbs**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/shockbs.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
     },
     "record": {
         "A": ["69.30.249.53"],
-        "MX": ["mail.is-a.dev"]
+        "CNAME": ["proxy.private.danbot.host"]
```
Comment (Stef-00012) [inline domains/shockbs.json:8]: you can use CNAME with other records

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/shockbs.json (hunk 1 above)
Comment (shockbs) [reply domains/shockbs.json:8]: what do you mean

Comment (Stef-00012) [reply domains/shockbs.json:8]: > what do you mean

sorry, wrote wrong, you can't use CNAME with other records

Comment (Stef-00012) [reply domains/shockbs.json:8]: you should remove the A record if you want to use the CNAME

Comment (shockbs) [reply domains/shockbs.json]: ok

**Review by Stef-DP — APPROVED** [review]

**Review by Stef-DP — CHANGES_REQUESTED** [review]
has merge conflicts

File: domains/shockbs.json [hunk 2]
```diff
@@ -4,6 +4,6 @@
         "email": "apipherng@gmail.com"
     },
     "record": {
-        "A": ["69.30.249.53"]
+        "CNAME": ["proxy.private.danbot.host"]
```
Comment (Stef-00012) [inline domains/shockbs.json:7]: remove the [] from the CNAME

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/shockbs.json (hunk 2 above)
Comment (shockbs) [reply domains/shockbs.json]: done, \:D

**Review by Stef-DP — APPROVED** [review]

**Pull request merged**